nautobot/dcim/utils.py

```diff
@@ -53,13 +54,22 @@ def cable_status_color_css(record):
     """
     Given a record such as an Interface, return the CSS needed to apply appropriate coloring to it.
     """
+    CABLE_STATUS_TO_CSS_CLASS = {}
     if not record.cable:
         return ""
-    # The status colors are for use with labels and such, and tend to be quite bright.
-    # For this function we want a much milder, mellower color suitable as a row background.
-    base_color = record.cable.get_status_color().strip("#")
-    lighter_color = rgb_to_hex(*lighten_color(*hex_to_rgb(base_color), 0.75))
-    return f"background-color: #{lighter_color}"
+    else:
+        for choice in CableStatusChoices.CHOICES:
+            color_value = STATUS_COLOR_MAP[choice[1]]
+            if color_value == ColorChoices.COLOR_GREEN:
+                CABLE_STATUS_TO_CSS_CLASS[color_value] = "success"
+            elif color_value == ColorChoices.COLOR_AMBER:
+                CABLE_STATUS_TO_CSS_CLASS[color_value] = "danger"
+            elif color_value == ColorChoices.COLOR_CYAN:
+                CABLE_STATUS_TO_CSS_CLASS[color_value] = "info"
+            else:
+                continue
```

Comment:
Shouldn't need to run through this logic over again every time we call this function - can't it be set up once at init time? Or even just something like:

```
COLOR_TO_CSS_CLASS = {
    ColorChoices.COLOR_GREEN: "success",
    ColorChoices.COLOR_AMBER: "danger",
    ColorChoices.COLOR_CYAN: "info",
}
```